edge-modules/Relayer/Program.cs

```diff
@@ -9,13 +9,16 @@ namespace Relayer
     using Microsoft.Azure.Devices.Edge.ModuleUtil;
     using Microsoft.Azure.Devices.Edge.Util;
     using Microsoft.Extensions.Logging;
+    using TestResultCoordinator.Client;
 
+    using TestOperationResult = TestResultCoordinator.Client.TestOperationResult;
     /*
      * Module for relaying messages. It receives a message and passes it on.
      */
     class Program
     {
-        static readonly ILogger Logger = ModuleUtil.CreateLogger("Relayer");
+        const string ModuleId = "Relayer";
```

Comment:
ModuleId should be retrieved from Settings, it is added as an env variable IOTEDGE_MODULEID to all module